src/tools/sfdk/remoteprocess.cpp

```diff
@@ -240,7 +240,7 @@ void RemoteProcess::kill(const QString &signal, void (RemoteProcess::*callback)(
         m_killRunner.reset(nullptr);
     });
 
-    QString killCommand = QString("kill -%1 -- -%2 %2").arg(signal).arg(m_processId);
+    QString killCommand = QString("kill -%1 -%2 %2").arg(signal).arg(m_processId);
```

Comment:
Why is the process ID there twice, with and without a preceding minus?

Reply:
Negation of a PID results in killing the process group the process leads.

Reply:
Negation of a PID results in killing the process group the process leads.

Reply:
TIL :)